Sources/StringsLintFramework/Rules/Lint/JSONCommentRule.swift

```diff
@@ -87,12 +87,16 @@ extension JSONCommentRule: LintRule {
 
         DispatchQueue.concurrentPerform(iterations: self.declaredStrings.count) { index in
             let string = self.declaredStrings[index]
+            var severity = self.severity
             if let violationType = getViolation(for: string) {
                 lock.lock()
+                if ViolationType.missingScreenshotURL.reasonDescription == violationType.reasonDescription || ViolationType.invalidScreenshotURL.reasonDescription == violationType.reasonDescription {
```

Comment:
I think what would make more sense here and would be more robust going forward is to introduce a `violationSeverityOverride` value to the `ViolationType` enum that we can define an override value for for each different violation. Maybe something like this:
```swift
var severityOverride: ViolationSeverity? {
  switch self {
  case .invalidJSON:
    return nil
    <etc>
  }
}
```

and then in building the `Violation` type below, we can do something like:

```swift
severity: violation.severityOverride ?? self.severity
```

Reply:
Yep this makes a lot more sense. Just pushed